SU-SWS/stanford_paragraph_types
# Upgrades

# READY FOR REVIEW

# Summary
Upgrade path for those variants. Hoo boy does Drupal not like it when you change a field's enumerated values and there is content in the database. 

# Needed By (Date)
- EOD Thursday July 29th

# Urgency
- Critical, can't deploy without it. 

# Steps to Test

1. Refresh your local environment (code & db)
2. Check out this branch
3. Run `blt local:sync`
4. Run `drush updb -y`
5. Run `drush cim -y`
6. Run `drush csim local -y`
7. Run `drush features-import-all -y`

Review DB tables and existing entities for correct data.

# Associated Issues and/or People
- None, bug fixes.
- @nickmanning 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
